From 8108cbf56d8a21b8155a6bf92ce0c73593115d94 Mon Sep 17 00:00:00 2001
From: Stephen Breen <breen.machine@gmail.com>
Date: Thu, 19 Nov 2020 10:52:19 -0500
Subject: [PATCH 6/6] mod changelong

---
 debian/changelog.em | 2 --
 1 file changed, 2 deletions(-)

diff --git a/debian/changelog.em b/debian/changelog.em
index a86f1cf..4b8380c 100644
--- a/debian/changelog.em
+++ b/debian/changelog.em
@@ -6,8 +6,6 @@ Architecture: source
 Version: 1.8.9000-7bionic
 Distribution: bionic
 Urgency: high
-Maintainer: Dirk Thomas <dthomas\@osrfoundation.org>
-Changed-By: Dirk Thomas <dthomas\@osrfoundation.org>
 Description:
  ros-eloquent-gtest-vendor - The package provides GoogleTest.
 Changes:
-- 
2.25.1

